src/System.Net.Sockets/tests/FunctionalTests/Close.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Net.Sockets.Tests
+{
+    public class CloseTests
+    {
```

Comment:
This is already in dev/api, so to avoid future merge conflicts I'll leave it as-is for now.  If you feel strongly about this, please log an issue and I'll fix it after the two branches are reconciled.